distribution/lib/Standard/Test/0.0.0-dev/src/Shared/Round_Spec.enso

```diff
@@ -0,0 +1,288 @@
+from Standard.Base import all
```

Comment:
I don't think this is the right place to have this.

`Standard.Test` is a test library, like `JUnit`. We should not distribute our tests with it.

IMO we should just move this test altogether to `Table_Tests` if we want to run it for both scalars and Columns. I don't think there is much harm in moving the scalar case from `Tests` to `Table_Tests` - IMO much less than distributing our internal test suites as part of a general and user-facing testing library.